### DISCUSSION TITLE
Is there any API for upload/download files?

### DISCUSSION THREAD

**Discussion by zhuoju36** [Kitware/trame#226] [Q&A] [answered]
To visualize a model from the web UI, a common demand is to upload a model then render its data with a certain pipeline.
And sometimes we need to download some processed data or reports from the server. 
Is there (or will there be) any API to realize such function conveniently?

**Comment by jourdain** [comment, marked as answer]
Examples:
 - [Download](https://github.com/Kitware/trame/blob/master/examples/00_howdoi/download.py)
 - [Upload](https://github.com/Kitware/trame/blob/master/examples/00_howdoi/upload.py)

**Reply by zhuoju36** [reply]
Thanks!
